### ISSUE TITLE
IP Address Not Getting Blocked

### ISSUE THREAD

**Issue by Nzzyme** [AdguardTeam/AdguardFilters#14799]
***Description***: After blocking an IP address from the Filtering Log, it does not get blocked. After sometime, I am still seeing connections to that IP address.

* **Current behaviour**: IP Address not getting blocked even after the IP was blocked in AG.

* **Expected behaviour**: After blocking the IP address, future requests to that IP should be blocked.

**Screenshot**: http://image.ibb.co/gqAscS/Screenshot_20180415153110.jpg

**System**:
- Android version: 6.0.1 MMB29M
- MIUI version: MIUI Global 9.5 - Stable [ 9.5.1.0 (MHOMIEK) ]

**AdGuard Version**:
- AdGuard for Android 2.10.176

**Filters**:
English, Spyware, Social Media, Annoyances, Mobile Ads, Simplified domain names, EasyList China, Anti-Adblock Killer-Reek

**AdGuard Settings**:

- No user filter used
- Phishing & Malware Protection: Enabled
- Block ads in all apps: Enabled
- Search ads & self-promotion of sites: Disabled
- DNS Filtering enabled with DNSCrypt (Server: AdGuard DNS)
- HTTPS Filtering: Enabled
- High quality filtering: Enabled
- Filtering mode is Local VPN

**Comment by BlazDT** [comment]
Which rule did you used to block?

**Comment by Nzzyme** [comment]
||42.62.94.2$app=com.xiaomi.midrop

**Comment by BlazDT** [comment]
Rule looks correct. I guess our Android app is in Premium mode. Can you please try the latest beta AdGuard for Android beta version and if it is still not working, looks like a bug to me which should be reported here:

https://github.com/AdguardTeam/AdguardForAndroid/issues

**Comment by Nzzyme** [comment]
I installed the latest beta (2.11.80 RC) and observed that the Filtering Log now does not have the enable/disable button. Not sure why was that removed :( Since this feature takes RAM & more battery, we could turn it off earlier.

Anyway, since I installed the latest beta, I am not seeing the entry at all in the log whereas it should ideally show even the blocked entry highlighted in light red?

**Comment by BlazDT** [comment]
Normally it should be displayed in log if the request is made. The button will return if I recall.
Any news?

**Comment by Nzzyme** [comment]
Yea, I thought so but strangely from the time I installed the latest beta, I am not seeing that entry (with IP address) at all in the filtering log. Instead, I am seeing domain names in the log. I removed the rule (which I posted above) from the user filter and surprisingly, not seeing that entry in the filtering log.

Do you recollect any change in AG in terms of processing IP addresses?

Thanks!

**Comment by Alex-302** [comment]
try 
`42.62.94.2$network`

**Comment by Nzzyme** [comment]
Today, I saw this IP address in the Filtering Log again and I applied the new rule provided by Alex and the request was blocked.

Does this mean that for IP addresses seen in the Filtering Log, this format should always be followed:

xx.xx.xx.xx$network

This should then be included in the "Options" section for IP addresses after the user taps on Block in the Filtering Log.. I will raise a enhancement request for this. Thank you!
